app/controllers/activitypub/inboxes_controller.rb

```diff
@@ -5,23 +5,24 @@ class ActivityPub::InboxesController < Api::BaseController
   include JsonLdHelper
   include AccountOwnedConcern
 
+  before_action :skip_unknown_actor_delete
+  before_action :require_signature!
+
   def create
-    if unknown_deleted_account?
-      head 202
-    elsif signed_request_account
-      upgrade_account
-      process_payload
-      head 202
-    else
-      render plain: signature_verification_failure_reason, status: 401
-    end
+    upgrade_account
+    process_payload
+    head 202
   end
 
   private
 
+  def skip_unknown_actor_delete
+    head 202 if unknown_deleted_account?
+  end
```

Comment:
Will this prevent other hooks and methods from running and returning an error?

Reply:
(Also, everything in this file is orthogonal to the purpose of this PR)

Reply:
When a `before_action` returns a response (e.g. `redirect_to` or `render` or `head`) it stops the execution chain